platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/UserResolver.java

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.git;
+
+import org.eclipse.che.api.git.shared.GitUser;
+
+/**
+ * Resolves git user.
+ *
+ * @author Max Shaposhnik
+ *
+ */
+public interface UserResolver {
```

Comment:
I think that this name should be scoped more, it may confuse people if they meet that interface.
I propose to rename this interface to `GitUserResolver` and add appropriate javadoc with reference to `GitUser`.